.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,8 +1,15 @@
-**[optional] Jira ticket link**:
+<!--
+Hi there. Thanks for submitting a pull request!
 
-Changes proposed in this pull request:
+Please describe your changes in this PR description.
+You can also include a link to an active Jira ticket that provides relevant context if there is one.
 
-*
-*
-*
-*
+https://policies.zephyrai.bio/sdlc.html
+-->
+# Proposed Changes
+
+<!--
+**Jira ticket**: https://zephyrai.atlassian.net/browse/TICKET
+-->
+
+<!-- Don't forget to motivate your pull request! -->
```

Comment:
what does motivate mean?

Reply:
comes from [motivating an argument or thesis](https://www.brandeis.edu/writing-program/resources/students/academic/motivating-the-argument.html). basically, explain why your PR is important, relevant, necessary. convince the reader that it's worth reviewing and worth merging.

Reply:
Could swap to `describe`, I'm pretty indifferent.